Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+# What is an Inclusive Minute
```

Comment:
I think the title might be better as "Introducing Your Newest Meeting Feature - The Inclusive Minute" or something that makes it clear that it relates to something you do during a meeting. And then the first heading can be "what is an inclusive minute?"

Reply:
Done in 47b11b940c0afb01e89e481e34a6244b64f1a45d